service/src/main/kotlin/fi/espoo/evaka/placement/PlacementService.kt

```diff
@@ -273,6 +274,40 @@ private fun checkAclAuth(aclAuth: AclAuthorization, placement: Placement) {
     }
 }
 
+data class UnitChildrenCapacityFactors(
+    val childId: PersonId,
+    val assistanceNeedFactor: Double,
+    val ageFactor: Double
+)
+
+fun Database.Read.getUnitChildrenCapacities(
+    unitId: DaycareId,
+    date: LocalDate
+): List<UnitChildrenCapacityFactors> {
+    return this.createQuery(
+        """
+        SELECT
+            ch.id child_id,
+            MAX(COALESCE(an.capacity_factor, 1)) assistance_need_factor,
+            MAX(CASE
+                WHEN u.type && array['FAMILY', 'GROUP_FAMILY']::care_types[] THEN 1.75
+                WHEN extract(YEARS FROM age(ch.date_of_birth)) < 3 THEN 1.75
+                ELSE 1.0
+            END) AS age_factor
```

Comment:
This other factor is affected not only by age and unit but also by service need. So the `ELSE` branch should use `service_need_option`'s `occupancy_coefficient`.

Reply:
Changed this to `coalesce(sno.occupancy_coefficient, default_sno.occupancy_coefficient, 1)`